browser-extension/src/addVideoButtons.js

```diff
@@ -159,5 +159,14 @@ function addRateButtons() {
         });
       },
     });
+
+    addVideoButton({
+      id: 'tournseol-watch-button',
+      label: 'Watch on Tournesol',
+      iconSrc: chrome.runtime.getURL('images/watch.svg'),
+      onClick: () => {
+        window.open(`${frontendUrl}/video/${videoId}`, '_blank');
```

Comment:
The URL `/video/{video_id}` is a legacy URL we don't want to use anymore. We can use the new URL instead:

```suggestion
        window.open(`${frontendUrl}/entities/yt:${videoId}?utm_source=extension&utm_medium=video_button`, '_blank');
```

I also added a couple query parameters to make the team able to see how users interact with the browser extension (those stats are private and anonymous).